app/controllers/courses_controller.rb

```diff
@@ -70,6 +70,10 @@ def update
     end
     params[:course].delete(:course_length_other_length)
 
+    # Remove commas from fees
```

Comment:
I think this comment is redundant, it's sufficiently obvious to me from the following code